Workflow.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:30:17 2016
+
+@author: mina
+"""
+
+import yaml
+
+class Workflow:
+    def __init__(adress,self):      
+      with open(adress, 'r') as f:
+          doc = yaml.load(f)
```

Comment:
What keys do we want to have in self.config. variant, workflow, stage, host_type ?